src/pkt_proc.c

```diff
@@ -1112,7 +1112,7 @@ void* process_packet (unsigned char *ctx_ptr,
 	    }
             return NULL;
         }
-        record->invalid++;
+        record->invalid = 1;
```

Comment:
This is similar to code that @smagill has reviewed before. (relevant tags: process, packet)